### DISCUSSION TITLE
impossible de graver des images supérieures à 40mmX40mm

### DISCUSSION THREAD

**Discussion by hervedetocane** [arkypita/LaserGRBL#2486] [General]
Bonjour j'ai un sculpfun S9 et je n'arrive pas à graver ou couper des objets plus grands que 40X40mm alors que la surface de travail est en principe de 40X40CM.
Merci pour votre aide.
Hervé

**Comment by Tribo-2dot0** [comment]
Verifique se nas Configurações Grbl $130 e $131 estão corretamente configurados em mm ($130=400 e $131=400)
